server/views/pages/ccardIndex.njk

```diff
@@ -37,8 +37,79 @@
 
             {% if calculationHistory.length %}
                 {% if latestCalculationCardConfig %}
-                    <h3 class="govuk-heading-m">Latest calculation</h3>
-                    {{ latestCalculationCard(latestCalculationCardConfig, latestCalculationCardAction) }}
+                    {% set latestCalc = calculationHistory[0] %}
```

Comment:
there's quite a few nested conditions here , it may be neater if we lift out this block of code into a partial or macro?